litellm/proxy/auth/handle_jwt.py

```diff
@@ -107,6 +115,18 @@ def get_user_id(self, token: dict, default_value: Optional[str]) -> Optional[str
             user_id = default_value
         return user_id
 
+    def get_user_email(
+        self, token: dict, default_value: Optional[str]
+    ) -> Optional[str]:
+        try:
+            if self.litellm_jwtauth.user_email_jwt_field is not None:
+                user_email = token[self.litellm_jwtauth.user_email_jwt_field]
+            else:
+                user_email = None
+        except KeyError:
+            user_email = default_value
```

Comment:
why are we returning a default value for user_email? 

will all users use the same email?

Reply:
![image](https://github.com/BerriAI/litellm/assets/3780301/f87a2e22-cbfe-46f5-ba9e-a6f0b7e5ef17)
Using the same logic for user_id - default_value is None